locking/locks.go

```diff
@@ -318,33 +324,26 @@ func (c *Client) lockIdFromPath(path string) (string, error) {
 	}
 }
 
-// Fetch locked files for the current committer and cache them locally
+// Fetch locked files for the current user and cache them locally
 // This can be used to sync up locked files when moving machines
 func (c *Client) refreshLockCache() error {
-	// TODO: filters don't seem to currently define how to search for a
-	// committer's email. Is it "committer.email"? For now, just iterate
-	locks, err := c.SearchLocks(nil, 0, false)
+	ourLocks, _, err := c.VerifiableLocks(0)
 	if err != nil {
 		return err
 	}
 
 	// We're going to overwrite the entire local cache
 	c.cache.Clear()
-
-	_, email := c.client.CurrentUser()
-	for _, l := range locks {
-		if l.Committer.Email == email {
-			c.cache.Add(l)
-		}
```

Comment:
@sinbad: I updated this to use `VerifiableLocks()`. Though now I'm wondering if the main `/locks` endpoint should also partition locks into `ours` and `theirs` slices?

IMO if we did that, we'd still want `VerifiableLocks()` specifically for `pre-push`, since it'll be where the server works out the more advanced lock workflows that we've been discussion.